lib/tool_shed/scripts/bootstrap_tool_shed/create_user_with_api_key.py

```diff
@@ -65,27 +67,10 @@ def create_api_key( app, user ):
     app.sa_session.add( new_key )
     app.sa_session.flush()
     return api_key
-    
+
+
 def create_user( app ):
-    user_info_config = os.path.abspath( os.path.join( app.config.root, 'lib/tool_shed/scripts/bootstrap_tool_shed', 'user_info.xml' ) )
-    email = None
-    password = None
-    username = None
```

Comment:
Okay. Can do. There were two equivalent implementations so I picked the cleanest one. I can see a case where it wouldn't set those variables, so I'll copy the lines.